src/main/java/tempest/ui/pages/ManageModulesPage.java

```diff
@@ -30,7 +30,7 @@ public ManageModulesPage(GUIManager guiManager) {
 
     @Override
     public String getName() {
-        return "manageModulesPage";
+        return PageNames.MANAGE_MODULES;
```

Comment:
This will stop the update check for the delete module button from working unless the name that it checks for is changed.

Reply:
> The checks pass now the name has been changed.

Thanks, I thought I'd checked for it everywhere.